controllers/backstage_extra_files.go

```diff
@@ -126,8 +128,11 @@ func (r *BackstageReconciler) extractExtraFileNames(ctx context.Context, backsta
 			// Limit to that file only
 			files = append(files, secExtraFile.Key)
 		} else {
-			sec := v1.Secret{}
-			if err = r.Get(ctx, types.NamespacedName{Name: secExtraFile.Name, Namespace: ns}, &sec); err != nil {
+			sec, err := r.Clientset.CoreV1().Secrets(ns).Get(ctx, secExtraFile.Name, metav1.GetOptions{})
+			if err != nil {
+				if errors.IsForbidden(err) {
+					return nil, fmt.Errorf("mounting extra files based on secrets without a key specified is not enabled for namespace %s: %s", ns, err)
```

Comment:
Can the error message indicate what users should do? Like `Either specify a key for each `spec.application.extraFiles[].secrets[]` or contact your administrator to grant secret read permission to the operator service account`?